src/view/frontend/tailwind/tailwind.config.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Since Hyvä Themes 1.1.14 this file is automatically
+ * merged into your Hyvä Theme Tailwind config
+ *
+ * See https://docs.hyva.io/hyva-themes/compatibility-modules/tailwind-config-merging.html
+ *
+ **/
+
+module.exports = {
+    purge: {
```

Comment:
Two concerns.

1. Need to include phtml files also in the purge list.
2. What if the theme use version 3? then `purge.content` wont work. Only `content` path exist there I believe.

Reply:
phtml is already included with the new mechanism, it also discovers both Tailwind version 2 and 3 and automatically adjusts the purge object.